### PR TITLE
Feature/bot

### PR DESCRIPTION
**Pull request by yawnka**
Jira Story Card / Issue Link
Spring 3 Mascot Bot Functionality

Description of PR:
Added the mascot bot with CSS that displays different dad jokes/affirmations/etc.
bot text from: https://docs.google.com/spreadsheets/d/16BrPG5-ioTw4L0QnN5dvOdazU_D5sSp5hhp-A4glgPQ/edit#gid=4148052

Type of change in this PR:
added a js page for functionality
added code of the fx ^ into home
added CSS code in tailwind css
Testing instructions:
npm start and click the starfish to get randomized output.

What I learned/had trouble with:
Functions, getting the srv image to display, CSS positioning.

### CODE REVIEW

**Review by PaulaTam — CHANGES_REQUESTED** [review]
- The Home.js needs to be re-edited to have the deleted stuff re-added as that is the current dashboard grid css.
- Some of the css in tailwind.css is the old css. I'll mark the parts I know with a comment.

File: src/pages/Home.js [hunk 1]
```diff
@@ -1,52 +1,47 @@
...
+import GenerateMessage from '../js/botPrompt.js';
 
 const Home = () => {
+  const displayMascot = () => {
+    return <GenerateMessage />;
+  };
+
   return (
     <main>
-      <div className="dashboard-grid">
```
Comment (PaulaTam) [inline src/pages/Home.js:12]: - The deleted parts should be brought back as this is the current css.

File: src/pages/Home.js [hunk 2]
```diff
@@ -1,52 +1,47 @@
...
-                    <Link to="/free-journal">Free Write</Link>
-                  </button>
-                  <br />
-                  <button>
-                    <Link to="/prompt-journal">Random Prompt</Link>
-                  </button>
-                </div>
-              )}
-            </Popup>
+      <h1 className="text-3xl font-bold underline">Title</h1>
```
Comment (PaulaTam) [inline src/pages/Home.js:16]: - This part is old code, please remove.

File: src/pages/Home.js [hunk 3]
```diff
@@ -1,52 +1,47 @@
...
-        </div>
+        )}
+      </Popup>
+      <br />
+      <br />
+      <div>
+        <DisplayJournal />
       </div>
-      <footer>This is a temp footer</footer>
+      {displayMascot()}
```
Comment (PaulaTam) [inline src/pages/Home.js:44]: The displayMascot() is fine where it is.

File: src/styles/tailwind.css [hunk 4]
```diff
@@ -116,14 +116,50 @@ body {
   .list-wrapper{
     @apply max-w-[382px] m-3 p-4 border-[1px] rounded-lg border-custom-burnt-orange bg-custom-cream
   }
+}
 
-  .task-wrapper{
```
Comment (PaulaTam) [inline src/styles/tailwind.css:120]: This task-wrapper (with the @apply) is the correct css.

File: src/styles/tailwind.css [hunk 5]
```diff
@@ -116,14 +116,50 @@ body {
...
+.button {
+  cursor: pointer;
+}
 
+.container {
+  text-align: center;
+  margin-top: 50px;
 }
 
-.Collapsible__trigger{
```
Comment (PaulaTam) [inline src/styles/tailwind.css:126]: This collapsible-trigger needs to be brought back. 

Just add a closing bracket at the end
`.Collapsible__trigger{
  color: #0E8992
}`

**Comment by yawnka** [comment]
old PR doesnt comply with updated branch, code used in feature/botFix

**Pull request closed without merging**